tests/models/granite_speech/test_modeling_granite_speech.py

```diff
@@ -338,9 +338,11 @@ def _load_datasamples(self, num_samples):
         return [x["array"] for x in speech_samples]
 
     @slow
-    @pytest.mark.skip("Public models not yet available")
+    @pytest.mark.skipif(not is_peft_available(), reason="Outputs diverge without lora")
```

Comment:
I don't understand very well here. Could you explain what are the different situations when `peft` is in the system and not in the system, it will cause this code in this test doing something differently and output will be different?